include/dpp/message.h

```diff
@@ -569,6 +569,8 @@ struct CoreExport attachment {
 	uint32_t height;
 	/** MIME type of the attachment, if applicable */
 	std::string content_type;
+	//** Whether this attachment is ephemeral, if applicable */
+	bool ephemeral;
```

Comment:
you will need to make sure to initialise this value in attachment::attachment (the constructor) to false.